check_process

```diff
@@ -1,12 +1,17 @@
-;; Test Rainloop
-    auto_remove=1
-    ; Manifest
+# See here for more information
+# https://github.com/YunoHost/package_check#syntax-check_process-file
+
+# Move this file from check_process.default to check_process when you have filled it.
```

Comment:
Those comment lines can be removed.

Reply:
I usually keep comments because it's easier to update a package from example_ynh with the comments. ^^ (using winmerge)